gen/dcos-config.yaml

```diff
@@ -1799,6 +1799,9 @@ package:
               {
                   "Command": ["iptables-save"]
               },
+              {
+                  "Command": ["sysctl", "-a"]
```

Comment:
Can you add a test for this command? Just add a `sysctl_-a` [here](https://github.com/dcos/dcos/blob/bc3ff6568a3b8ea552bbb6bc5a43bed5b2989079/packages/dcos-integration-test/extra/test_dcos_diagnostics.py#L450-L471)